staff/doboapps/js/challenges/to-camel-case/scripts/toCamelCase.js

```diff
@@ -0,0 +1,22 @@
+function toCamelCase(str) {
+
+    if (typeof str !== 'string')
+    throw Error('input str is not a string');
+
+
+    var sentence = "";
+    str = str.toLowerCase();
+    
+    var myArray = str.split(" ");
+
+    myArray.map(word => {
```

Comment:
=>????.... this is ES5! 

<img src="https://media2.giphy.com/media/2BG086WOP2Xfi/giphy.gif" />